wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -546,6 +546,7 @@ test-suite wallet-new-specs
                     MarshallingSpec
                     SwaggerSpec
                     RequestSpec
+                    DecryptSpec
```

Comment:
Because you asked, here's my two cents:

Best is to organize tests using the same path structure than the source code to make the mapping obvious.
So: Cardano.Wallet.Kernel.DecryptSpec would be better. 

Though.. I admit it's a bit of a mess currently in the wallet and we (a.k.a the whole cardano dev team) haven't really settled on a structure. Hence, I am fine with `DecryptSpec` as it is.